components/EditDeleteDropDown.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import Image from "next/image";
+export default function EditDeleteDropDown({ onDropDownChange, className }) {
+  const dropdown = `w-[24px] h-[24px] relative`;
+  const dropdownMenuList = `w-[139px] h-[92px] absolute top-[24px] right-0 border-[1px] border-d1d5d8 rounded-[8px]`;
+  const dropdownMenu = `w-[139px] h-[46px] flex justify-center items-center text-[16px] text-6b7280`;
+
+  const [isOpen, setIsOpen] = useState(false);
+  const [seletedItem, setSeletedItem] = useState(null);
+  const toggleDropDown = () => setIsOpen((prev) => !prev);
+  const handleItemClick = (item) => {
+    setSeletedItem(item);
+    setIsOpen(false);
+    onDropDownChange(item);
+  };
+  const items = ["수정하기", "삭제하기"];
```

Comment:
@taeyeonkim94 이렇게 하기보다는 아래처럼 하는게 더 좋을거같아요.
이유는 아래 코멘트와 함께 생각해보세요
```suggestion
const items = [
  {key: 'edit', field: '수정하기'},
  {key: 'delete', field: '삭제하기'},
]
```